### ISSUE TITLE
document4j - Pdf to Docx conversion (Converted DocX) linkable is not available in table of content 

### ISSUE THREAD

**Issue by dkaruppan** [documents4j/documents4j#109]
I am evaluting document4j  for converting Pdf to DocX after converting I am haviing following clarfificarion.

1. I have converted Pdf to docx but converted document doesn't having linkable option.IS that able to fix by document4j team or any other solution ?

2. THis document4j converter suitable for windows os or we can use in linux also?

**Comment by raphw** [comment]
1. It uses the VBS script that is part of the project. You can modify that script and override it via the command line. If you can adjust the VBS script, it can certainly be done.
2. You can use it on Linux but not converters that require Windows such as all MS-product based converters.

**Comment by waqasraz** [comment]
How can we override the scripts on the server?

**Comment by raphw** [comment]
Yes, you can create a custom VBS script and supply it via the command line: https://github.com/documents4j/documents4j/blob/master/documents4j-transformer-msoffice/documents4j-transformer-msoffice-word/src/main/java/com/documents4j/conversion/msoffice/MicrosoftWordScript.java#L51

**Comment by waqasraz** [comment]
I did some changes to a word-convert file and running the server like this, passing folder with 4 VBS files.

```
 java -Dcom.documents4j.conversion.msoffice="C:\Users\user1\Downloads\convert" -jar documents4j-server-standalone-1.1.5-shaded.jar --level debug http://0.0.0.0:9998
```

But it does not seem to be working. Any idea?

**Comment by waqasraz** [comment]
I think I was able to make it work. 
```
 java -Dcom.documents4j.conversion.msoffice.word_convert.vbs="C:\Users\user1\Downloads\convert\word_convert.vbs" -jar documents4j-server-standalone-1.1.5-shaded.jar --level debug http://0.0.0.0:9998
```
